lms/static/js/spec/bookmarks/bookmarks_list_view_spec.js

```diff
@@ -125,25 +152,114 @@ define(['backbone', 'jquery', 'underscore', 'logger', 'js/common_helpers/ajax_he
 
                 expect(bookmarksButtonView.bookmarksListView.$('.bookmarks-empty-detail-title').text().trim()).
                     toBe(emptyListText);
+
+                expect(bookmarksButtonView.bookmarksListView.$('.paging-header').length).toBe(0);
+                expect(bookmarksButtonView.bookmarksListView.$('.paging-footer').length).toBe(0);
             });
 
             it("has rendered bookmarked list correctly", function () {
                 var requests = AjaxHelpers.requests(this);
-                var url = BOOKMARKS_API_URL + '?course_id=COURSE_ID&page_size=500&fields=display_name%2Cpath';
-                var expectedData = createBookmarksData(3);
+                var expectedData = createBookmarksData({numBookmarksToCreate: 3});
 
-                spyOn(bookmarksButtonView.bookmarksListView, 'courseId').andReturn('COURSE_ID');
                 bookmarksButtonView.$('.bookmarks-list-button').click();
                 expect($('#loading-message').text().trim()).
                     toBe(bookmarksButtonView.bookmarksListView.loadingMessage);
 
-                AjaxHelpers.expectRequest(requests, 'GET', url);
+                verifyRequestParams(
+                    requests,
+                    {course_id: 'a/b/c', fields: 'display_name,path', page: '1', page_size: '10'}
+                );
                 AjaxHelpers.respondWithJson(requests, expectedData);
 
                 expect(bookmarksButtonView.bookmarksListView.$('.bookmarks-results-header').text().trim()).
                     toBe('My Bookmarks');
 
                 verifyBookmarkedData(bookmarksButtonView.bookmarksListView, expectedData);
+
+                expect(bookmarksButtonView.bookmarksListView.$('.paging-header').length).toBe(1);
+                expect(bookmarksButtonView.bookmarksListView.$('.paging-footer').length).toBe(1);
+            });
+
+            it("calls bookmarks list render on page_changed event", function () {
```

Comment:
Got it. Still though, if this code were using the `list.js` component, this functionality would already work (and is already tested).